app/Presenters/templates/Homepage/templateHowDoI.latte

```diff
@@ -0,0 +1,160 @@
+{block content}
+    <div class="container">
+        <img src="{$basePath}/img/logo.svg" id="logo">
+        <div class="center">
+            <p class="claim">Dev-stack solution based on Nette framework, which provides you with fully ready application skeleton
+                and environment <strong>to start developing a web application.</strong>
+            </p>
+            <pre>$ composer create-project instante/skeleton</pre>
+        </div>
+    </div>
+    <div class="blueline">
+        <div class="container center">
+            <div id="searchbox">
```

Comment:
There shouldn't be multiple elements of the same ID in a document, please use only CSS classes for styling.